### DISCUSSION TITLE
[QUESTION]  Reading or converting the JSON file

### DISCUSSION THREAD

**Discussion by lord666** [Andrewthe13th/Inventory_Kamera#332] [General]
Hello, 
First, Thank you, because your software is really excellent and very practical. 

I would like to know, if it's possible to use the "Json" data from "Inventory Kamara" in another software like Excel ? I tried, but apparently the current JSON format doesn't seem supported. If not, is there a way to convert them to another format? 

Thanks in advance

**Comment by Cupcak3** [comment]
Kamera exports in a format called GOOD that is a modified form of JSON so importing into softwares that aren't built with GOOD in mind will likely not work 100% out of the box as you expect.
